packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -25,6 +25,7 @@ export class Choice extends React.PureComponent {
       );
     }
 
+    this.input = null;
```

Comment:
`this.input` was already used internally in `Choice`, I just defined it here for extra clarity